lib/cosmos/top_level.rb

```diff
@@ -162,11 +162,12 @@ def self.data_path(name)
       # No Gemfile - so no gem based extensions
     end
 
+    # Check CORE
     filename = File.join(::Cosmos::PATH, 'data', name)
     return filename if File.exist? filename
 
-    # Check CORE
-    filename = Cosmos.path('data', name)
+    # Check relative to executing file
+    filename = Cosmos.path($0, 'config/data/' + name)
```

Comment:
No.  The idea with this was to be able to find config/data files in a COSMOS gem based tool during development (or installation not as a gem).  DMT is an example with finding its icon.